tracer/test/test-applications/integrations/Samples.AWS.Lambda/Samples.AWS.Lambda.csproj

```
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

We have to use `<TargetFrameworks>` here to override the values set from the parent Directory.Build.props

```suggestion
    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
```